python/lsst/pipe/tasks/postprocess.py

```diff
@@ -657,34 +665,32 @@ def run(self, parq, funcs=None, dataId=None):
         dfDict = {}
         analysisDict = {}
         templateDf = pd.DataFrame()
-        # Perform transform for data of filters that exist in parq and are
-        # specified in config.filterMap
-        for filt in parq.columnLevelNames['filter']:
-            if filt not in self.config.filterMap:
-                self.log.info("Ignoring %s data in the input", filt)
+        # Perform transform for data of filters that exist in parq.
+        for band in parq.columnLevelNames['band']:
+            if self.config.outputBands != [] and band not in self.config.outputBands:
```

Comment:
I think the code would be a bit cleaner if you turned `outputBands` into the set of bands you're actually going with, and then used that consistently, instead of constantly referring back to the raw config.